Expand Difficulty Increase mod articles (#12629)
ppy/osu-wiki: wiki/Gameplay/Game_modifier/Perfect_(lazer)/en.md

*For the osu!stable version of this article, see: [Perfect (mod)](/wiki/Gameplay/Game_modifier/Perfect)*\
*For the full list of all [lazer](/wiki/Client/Release_stream/Lazer) mods, see: [Game modifier (lazer)](/wiki/Gameplay/Game_modifier_(lazer))*

> The **Perfect** mod automatically restarts the [beatmap](/wiki/Beatmap) if you are no longer getting an SS grade. This includes missing a note, a [spinner](/wiki/Gameplay/Hit_object/Spinner), a [slider tick](/wiki/Gameplay/Hit_object/Slider/Slider_tick), a [slidertail](/wiki/Gameplay/Hit_object/Slider/Slidertail), and getting something other than a GREAT from any object. 

Reviewer: ```suggestion
The **Perfect** mod automatically restarts the [beatmap](/wiki/Beatmap) if the player no longer can get an SS grade. This includes missing a note, a [spinner](/wiki/Gameplay/Hit_object/Spinner), a [slider tick](/wiki/Gameplay/Hit_object/Slider/Slider_tick), a [slidertail](/wiki/Gameplay/Hit_object/Slider/Slidertail), or getting something other than a GREAT from any object. 
```

## Customisation

![Perfect customisation options in the game client](/wiki/Gameplay/Game_modifier_(lazer)/img/customise/PF.jpg)

> - **Restart on fail** (default enabled): Restart the map after the [health bar](/wiki/Client/Interface/Health_bar) hits the bottom. *Enabling this option doesn't make the mod unrankable.*

Reviewer: ```suggestion
- **Restart on fail** (default: enabled): Restart the map after the [health bar](/wiki/Client/Interface/Health_bar) hits the bottom. *Enabling this option doesn't make the mod unrankable.*
```
